web/templates/popovers/user_group_info_popover.hbs

```diff
@@ -28,7 +28,7 @@
                 <span class="popover-group-menu-placeholder"><i>{{t 'This group has no members.'}}</i></span>
             {{/if}}
         </li>
-        {{#unless (or is_guest is_system_group)}}
+        {{#unless (or is_guest is_system_group deactivated)}}
```

Comment:
This makes sure that we do not show "Group settings" for deactivated groups in the popover. We can further add some indicator to show that the group is deactivated.

Reply:
Yeah, noting it the group popover sounds good. Probably an italicized "This group has been deactivated." line just below the group description would work well, similar to the user popover:

![Screenshot 2024-09-16 at 12 10 25@2x](https://github.com/user-attachments/assets/da492a9f-8f5e-411c-95e8-6c25e1a976f8)

Reply:
Added this. This is how the popover looks for groups with and without description -
![Screenshot from 2024-09-17 11-57-49](https://github.com/user-attachments/assets/fd478638-3f05-4038-8b4a-e23a1d35b3f5)
![Screenshot from 2024-09-17 11-57-37](https://github.com/user-attachments/assets/821cbf10-5f5d-4c7a-8bd8-4fba2f43aac7)

Reply:
Works for me! Separately, we should probably bump up the contrast on that line a bit, at least in light theme.